test/controllers/architectures_controller_test.rb

```diff
@@ -11,16 +11,6 @@ class ArchitecturesControllerTest < ActionController::TestCase
   basic_pagination_per_page_test
   basic_pagination_rendered_test
 
-  def test_new_submit_button_id
```

Comment:
These three tests aren't useful at all, the proper rendering is tested by `basic_new_test` and `basic_edit_test`